CHANGELOG.md

```diff
@@ -4,7 +4,8 @@ As this project is pre 1.0, breaking changes may happen for minor version bumps.
 
 ## Unreleased
 
-* Fixed missing Liquidity Pool ID in AccountResponse Balance ([#379](https://github.com/stellar/java-stellar-sdk/pull/379)).
+* Fix missing Liquidity Pool ID in AccountResponse Balance ([#379](https://github.com/stellar/java-stellar-sdk/pull/379)).
+* Fix null pointer when ChangeTrustOperationResponse.getAsset for LiquidityPool trust line ([#378](https://github.com/stellar/java-stellar-sdk/pull/378)).
```

Comment:
I think we should insert the word  "calling" in between "when" and  "ChangeTrustOperationResponse.getAsset" :

Fix null pointer when calling ChangeTrustOperationResponse.getAsset for LiquidityPool trust line

Reply:
[62783a3](https://github.com/stellar/java-stellar-sdk/pull/380/commits/62783a3a44514605cd2934b694f36900f1692088)